delfin/drivers/dell_emc/vmax/constants.py

```diff
@@ -26,3 +26,144 @@
                  "WriteResponseTime"
                  ]
 VMAX_REST_TARGET_URI_ARRAY_PERF = '/performance/Array/metrics'
+
+VMAX_METRICS = [
+    'iops',
+    'readIops',
+    'writeIops',
+    'throughput',
+    'readThroughput',
+    'writeThroughput',
+    'responseTime'
+]
+
+BEDIRECTOR_METRICS = {
+    'iops': 'IOs',
+    'throughput': 'MBs',
+    'readThroughput': 'MBRead',
+    'writeThroughput': 'MBWritten',
+}
+FEDIRECTOR_METRICS = {
```

Comment:
Dont't we have other metrics from FE diretor?

Reply:
Below metrics should be available for FE ports, please check
Reads-> readIops
Writes-> writeIops
MBRead- read throughput
MBWritten
ResponseTime
ReadResponseTime
WriteResponseTime

Reply:
https://powermax-for-splunk.readthedocs.io/_/downloads/en/latest/pdf/ this document has metric list of vmax/powermax